src/coreclr/vm/arm64/asmhelpers.asm

```diff
@@ -345,6 +349,8 @@ wbs_card_bundle_table
         DCQ 0
 wbs_GCShadow
         DCQ 0
+wbs_GCShadowEnd
+        DCQ 0
```

Comment:
Could you please move `wbs_GCShadow` and `wbs_GCShadowEnd` to the end of the wbs block? These are debug-only fields, having them at the end would be a tiny bit better for cache locality.

Reply:
@jkotas moved to the end of the wbs